charts/agent-operator/templates/operator-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ga-operator.fullname" . }}
+  labels:
+    app.kubernetes.io/component: operator
+    {{- include "ga-operator.labels" . | nindent 4 }}
+  {{- with .Values.annotations }}
+  annotations:
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "ga-operator.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+{{ include "ga-operator.labels" . | indent 8 }}
+{{- with .Values.podAnnotations }}
+      annotations:
+{{ toYaml . | indent 6 }}
+{{- end }}
+    spec:
+      {{- if .Values.serviceAccount.create }}
+      serviceAccountName: {{ template "ga-operator.serviceAccountName" . }}
+      {{- end }}
```

Comment:
The operator will definitely always need a service account, right? If the user doesn't create one, should we have a way of defining a custom one?

Reply:
good call, removed the `if` clause